### ISSUE TITLE
Reason for calling fit thrice in train.py

### ISSUE THREAD

**Issue by VeeranjaneyuluToka** [yuto3o/yolox#15]
Hi, thanks for nice code!

I was trying to understand the reason for calling fit method thrice in train.py, would you mind explaining a bit about it?

**Comment by yuto3o** [comment]
That's three different training strategies: warm-up training, freeze the backbone and finetune the whole model.

**Comment by VeeranjaneyuluToka** [comment]
Ok, got it. i just missed num variable that you initialized to differentiate between transfer learning and from scratch. Would you mind describing a bit about custom model?

**Comment by yuto3o** [comment]
The code is based on keras, you could modify the custom model as you like in https://github.com/yuto3o/yolox/blob/master/core/model/one_stage/custom/custom.py. Although Yolov4-tiny is available now, I customized my own model depending on Yolov4.

**Comment by VeeranjaneyuluToka** [comment]
Ok, thanks for info.
